src/encoded/tests/test_post_put_patch.py

```diff
@@ -208,9 +208,10 @@ def test_post_object_with_child(content_with_child, testapp):
         ]
     }
     res = testapp.post_json('/testing-link-targets', edit, status=422).json
-    assert len(res['errors']) == 1
-    assert res['errors'][0]['description'] == 'submission of calculatedProperty disallowed'
-    assert res['errors'][0]['name'] == 'Schema: reverse'
+    errors = sorted(res['errors'], key=lambda d: d['description'])
+    assert len(errors) == 2
+    assert errors[1]['description'] == 'submission of calculatedProperty disallowed'
+    assert errors[1]['name'] == 'Schema: reverse'
```

Comment:
Maybe something like this (untested):
```suggestion
    assert any(error['name'] == 'Schema: reverse' and error['description'] == 'submission of calculatedProperty disallowed'
               for error in res.json['errors'])
```